qa/os/src/test/java/org/elasticsearch/packaging/util/Packages.java

```diff
@@ -332,4 +306,41 @@ public static void restartElasticsearch(Shell sh, Installation installation) thr
         }
         assertElasticsearchStarted(sh, installation);
     }
+
+    /**
+     * A small wrapper for retrieving only recent journald logs for the
+     * Elasticsearch service. It works by creating a cursor for the logs
+     * when instantiated, and advancing that cursor when the {@code clear()}
+     * method is called.
+     */
+    public static class JournaldWrapper {
```

Comment:
Note: even though this is only invoked in one test on the master branch, I'm going to need it for keystore tests on the feature branch, and it seems like something that could be generally useful later on.